### ISSUE TITLE
[1.19.2] Another Jade error (NoSuchFieldError: component)

### ISSUE THREAD

**Issue by SimonMeskens** [hammertater/treechop#221]
This one seems to be distinct from the other one.

![image](https://github.com/hammertater/treechop/assets/4284735/4130db02-4862-406c-8e2e-b330ca764cf7)

Jade error output:

```
08/30/2023 - 18:43:28
ht.treechop.compat.Jade@6f50cefe
java.lang.NoSuchFieldError: component
	at TRANSFORMER/treechop@0.18.4/ht.treechop.compat.Jade.changeBlockName(Jade.java:70)
	at TRANSFORMER/treechop@0.18.4/ht.treechop.compat.Jade.appendTooltip(Jade.java:41)
	at TRANSFORMER/jade@8.9.0/snownee.jade.impl.BlockAccessorImpl._gatherComponents(BlockAccessorImpl.java:120)
	at TRANSFORMER/jade@8.9.0/snownee.jade.overlay.WailaTickHandler.tickClient(WailaTickHandler.java:160)
	at TRANSFORMER/jade@8.9.0/snownee.jade.util.ClientPlatformProxy.onClientTick(ClientPlatformProxy.java:138)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:260)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:252)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:315)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:296)
	at TRANSFORMER/forge@43.2.21/net.minecraftforge.event.ForgeEventFactory.onPostClientTick(ForgeEventFactory.java:839)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.Minecraft.m_91398_(Minecraft.java:1852)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1078)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.Minecraft.m_91374_(Minecraft.java:700)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.main.Main.m_239872_(Main.java:212)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.main.Main.main(Main.java:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at MC-BOOTSTRAP/fmlloader@1.19.2-43.2.21/net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.run(Launcher.java:106)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
```

**Comment by SimonMeskens** [comment]
Seems like that field is called `text` now, not `component`

**Comment by kadrinLef** [comment]
Im having this same issue

**Comment by SimonMeskens** [comment]
You will probably be able to solve the issue by going to an older version of Jade, but I haven't bothered seeing which version yet. If anyone finds the version, let us know in here, since I don't know if the author is going to want to fix an old version (the bug is fixed in 1.20)

**Comment by hammertater** [comment]
Hey, I'll get it fixed. 1.19.2 treechop is a lot more popular than 1.20 at the moment, and I've got other bug fixes coming out, so it's no trouble

**Comment by SimonMeskens** [comment]
Oh that's amazing, thank you so much!

**Comment by LunaciaDev** [comment]
> You will probably be able to solve the issue by going to an older version of Jade, but I haven't bothered seeing which version yet. If anyone finds the version, let us know in here, since I don't know if the author is going to want to fix an old version (the bug is fixed in 1.20)

Downgrading to Jade 8.8.1 for Forge 1.19.2 solve this issue for me, it start to happen from 8.9.0.
